feature/security/gnsi/acctz/tests/record_subscribe_full/record_subscribe_full_test.go

```diff
@@ -0,0 +1,828 @@
+}
+
+func setupUserPassword(t *testing.T, dut *ondatra.DUTDevice, username, password string) {
+	request := &credentialz.RotateAccountCredentialsRequest{
+		Request: &credentialz.RotateAccountCredentialsRequest_Password{
+			Password: &credentialz.PasswordRequest{
+				Accounts: []*credentialz.PasswordRequest_Account{
+					{
+						Account: username,
+						Password: &credentialz.PasswordRequest_Password{
+							Value: &credentialz.PasswordRequest_Password_Plaintext{
+								Plaintext: password,
+							},
+						},
+						Version:   "v1.0",
+						CreatedOn: uint64(time.Now().Unix()),
+					},
+				},
+			},
+		},
+	}
+
+	credzClient := dut.RawAPIs().GNSI(t).Credentialz()
+
+	credzRotateClient, err := credzClient.RotateAccountCredentials(context.Background())
+	if err != nil {
+		t.Fatalf("failed fetching credentialz rotate account credentials client, error: %s", err)
+	}
+
+	err = credzRotateClient.Send(request)
+	if err != nil {
+		t.Fatalf("failed sending credentialz rotate account credentials request, error: %s", err)
+	}
+
+	_, err = credzRotateClient.Recv()
+	if err != nil {
+		t.Fatalf("failed receiving credentialz rotate account credentials response, error: %s", err)
+	}
+
+	err = credzRotateClient.Send(&credentialz.RotateAccountCredentialsRequest{
+		Request: &credentialz.RotateAccountCredentialsRequest_Finalize{
+			Finalize: request.GetFinalize(),
+		},
+	})
+	if err != nil {
+		t.Fatalf("failed sending credentialz rotate account credentials finalize request, error: %s", err)
+	}
+
+	// brief sleep for finalize to get processed
+	time.Sleep(time.Second)
+}
+
+func setupUsers(t *testing.T, dut *ondatra.DUTDevice) {
+	auth := &oc.System_Aaa_Authentication{}
+	successUser := auth.GetOrCreateUser(successUsername)
+	successUser.SetRole(oc.AaaTypes_SYSTEM_DEFINED_ROLES_SYSTEM_ROLE_ADMIN)
+	auth.GetOrCreateUser(failUsername)
+	ondatragnmi.Update(t, dut, ondatragnmi.OC().System().Aaa().Authentication().Config(), auth)
+	setupUserPassword(t, dut, successUsername, successPassword)
+}
+
+func dialGRPC(t *testing.T, target string) (*grpc.ClientConn, net.Addr) {
+	var addrObj net.Addr
+
+	conn, err := grpc.Dial(
+		target,
+		grpc.WithTransportCredentials(
+			credentials.NewTLS(
+				&tls.Config{
+					InsecureSkipVerify: true,
+				},
+			),
+		),
+		grpc.WithContextDialer(func(ctx context.Context, a string) (net.Conn, error) {
+			dst, err := net.ResolveTCPAddr("tcp", a)
+			if err != nil {
+				return nil, err
+			}
+
+			c, err := net.DialTCP("tcp", nil, dst)
+			if err != nil {
+				return nil, err
+			}
+
+			addrObj = c.LocalAddr()
+
+			return c, err
+		}))
+	if err != nil {
+		t.Fatalf("failed grpc dialing %q, error: %v", target, err)
+	}
+
+	readyCounter := 0
+
+	for {
+		state := conn.GetState()
+
+		if state == connectivity.Ready {
+			break
+		}
+
+		readyCounter += 1
+
+		if readyCounter >= 10 {
+			t.Fatal("grpc connection never reached ready state")
+		}
+
+		time.Sleep(time.Second)
+	}
+
+	return conn, addrObj
+}
+
```

Comment:
there's a bunch of (I think) setup happening here, would it be better to pull the setup into distinct files outside of this file? keeping this file to just the tests you want to run?

At the least it might be nice to at least have some comments for each of these functions:
  // Setup test RPCs to be used in the acctz client tests

Reply:
Moved grpc/ssh helper functions to `internal/security/acctz/acctz.go` & fixed comments.